CHANGELOG/CHANGELOG-3.4.md

```diff
@@ -26,41 +24,32 @@ Previous change logs can be found at [CHANGELOG-3.3](https://github.com/etcd-io/
 - Improve [mvcc: reduce count-only range overhead](https://github.com/etcd-io/etcd/pull/15099)
 - Improve [mvcc: push down RangeOptions.limit argv into index tree to reduce memory overhead](https://github.com/etcd-io/etcd/pull/15137)
 - Improve [server: set multiple concurrentReadTx instances share one txReadBuffer](https://github.com/etcd-io/etcd/pull/15195)
+- Upgrade [github.com/grpc-ecosystem/grpc-gateway](https://github.com/grpc-ecosystem/grpc-gateway/releases) from [v1.9.5](https://github.com/grpc-ecosystem/grpc-gateway/releases/tag/v1.9.5) to [v1.11.0](https://github.com/grpc-ecosystem/grpc-gateway/releases/tag/v1.11.0).
+- Bump bbolt to [v1.3.7](https://github.com/etcd-io/etcd/pull/15223).
+- Fix [aligning zap log timestamp resolution to microseconds](https://github.com/etcd-io/etcd/pull/15241). Etcd now uses zap timestamp format: `2006-01-02T15:04:05.999999Z0700` (microsecond instead of milliseconds precision).
+- Fix [consistently format IPv6 addresses for comparison](https://github.com/etcd-io/etcd/pull/15188)
 
 ### Package `clientv3`
 - Fix [etcd might send duplicated events to watch clients](https://github.com/etcd-io/etcd/pull/15275).
 
-### Dependency
-- Upgrade [github.com/grpc-ecosystem/grpc-gateway](https://github.com/grpc-ecosystem/grpc-gateway/releases) from [v1.9.5](https://github.com/grpc-ecosystem/grpc-gateway/releases/tag/v1.9.5) to [v1.11.0](https://github.com/grpc-ecosystem/grpc-gateway/releases/tag/v1.11.0).
-- Bump bbolt to [v1.3.7](https://github.com/etcd-io/etcd/pull/15223).
-
-### Other
+### Docker image
 - Updated [base image from base-debian11 to static-debian11 and removed dependency on busybox](https://github.com/etcd-io/etcd/pull/15038).
 
-### Package `pkg/logutil`
```

Comment:
Packages outside of `clientv3` are not something we officially provide to users.